### PR TITLE
PAN-OS PCAPs from Panorama and threat-pcap

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20912
examples in issue

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation (with link to it)
- [x] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [CHANGELOG](https://github.com/demisto/content/blob/ab08ce9d703858897232845f533bc1f1a685c1e1/Integrations/Panorama/CHANGELOG.md)
- [ ] [YAML file](https://github.com/demisto/content/blob/ab08ce9d703858897232845f533bc1f1a685c1e1/Integrations/Panorama/Panorama.yml) - changes the description of the searchtime argument

### CODE REVIEW

File: Integrations/Panorama/Panorama.yml [hunk 1]
```diff
@@ -2232,8 +2232,8 @@ script:
       required: false
       secret: false
     - default: false
-      description: The Search time for the request. For further information, see the
-        Panorama XML API Documentation.
+      description: 'The Search time for the request.  e.g: "2019/12/26 00:00:00",
```
Comment (kirbles19) [inline Integrations/Panorama/Panorama.yml]: For example: "2019/12/26 00:00:00", "2020/01/10". For more information, see the Panorama XML API documentation.'

**Comment by kirbles19** [comment]
@bakatzir Done.

**Comment by bakatzir** [comment]
fails on: Failed:`Endpoint Malware Investigation - Generic - Test` with integration(s): Traps,Cylance Protect v2,Demisto REST API failed
failing since night, yaakovi is looking at it, non related to pan-os.

**Comment by yuvalbenshalom** [comment]
Rechecked bars last comment. force merging.

**Pull request merged**